devansh2001/connect-me
# Issue 4 backend test end point

The requirement was to have the boilerplate code setup with the server running on the port 8080 the `/test` endpoint setup such that that endpoint returns `Hello, World!`

To test this:
* Switch branches to `issue-4-backend-test-end-point`
* Download or use the web version Postman or Insomnia and hit `http://localhost:8080/test` using the GET method
    * Postman and Insomnia are two tools that are generally used to test API endpoints. They both do the same thing
   * All you need to do is choose which method (GET, POST, PUT, DELETE, etc) and enter the URL. It will show you the response formatted in JSON style
    * Instead if you just want to use the terminal, you can run `curl "http://localhost:8080/test"`
* You should see `Hello, World!` response
